x/concentrated-liquidity/simulation/sim_msgs.go

```diff
@@ -143,22 +153,24 @@ func RandMsgCollectFees(k clkeeper.Keeper, sim *osmosimtypes.SimCtx, ctx sdk.Con
 		return nil, fmt.Errorf("user does not have pool tokens")
 	}
 
-	// perform swap until token 1 runs out
+	// perform 50 random swaps until token 0 runs out
 	remainingSwapOwnerToken0Amt := swapOwnerTokens[0].Amount
-	for remainingSwapOwnerToken0Amt.ToDec().TruncateInt().GT(sdk.ZeroInt()) {
+	swapsRemaining := 50
```

Comment:
@p0mvn also added this so that we only swap 50 time or until user's token0 runs out 

before we were swapping small amount for >100 times making simulator take a lot of time to run